src/components/_internal/BaseToastBanner/BaseToastBanner.types.ts

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { SpaceSize } from '../../../theme/space.types';
+import { InlineProps } from '../../layout/Inline/Inline';
+import { PadboxProps } from '../../layout/Padbox/Padbox';
+import { BaseToastBannerVariants } from './BaseToastBanner.enums';
+
+export type Variants = typeof BaseToastBannerVariants[keyof typeof BaseToastBannerVariants];
+
+export interface BaseToastBannerWrapperProps {
+  children: React.ReactNode;
+  variant: Variants;
+  paddingSize: PadboxProps['paddingSize'];
+  paddingType: PadboxProps['paddingType'];
+  stretch: InlineProps['stretch'];
+  size: SpaceSize;
```

Comment:
please rename to `iconSize` and change type to `number`. We shouldn't be using SpaceSize for widths and heights